README.md

```diff
@@ -49,7 +58,74 @@ The `fairlearn` package contains the following algorithms for mitigating unfairn
 > _Note_:
 > DP refers to demographic parity, EO to equalized odds, and BGL to bounded group loss. For more information on these and other terms we use in this repository please refer to the [terminology page](./TERMINOLOGY.md). To request additional algorithms or fairness definitions, please open a [new issue](https://github.com/fairlearn/fairlearn/issues).
 
-## Install fairlearn
+
+## Visualizations
+
+This repository containts a visualization dashboard that enables data scientists, auditors, business decision makers, and other stakeholders to visually observe and analyze how their model predictions are impacting different groups of data points (e.g., different ethnicities). It provides a high-level and detailed view of prediction insights for assessing which groups are negatively impacted. Using this dashboard you get insights into (un)fairness of you model and gain ideas around potential ways of mitigating the obsereved unfairness.  
+
+### Visualization: single model, pre-mitigation 
+
+Load the visualization dashboard in your noteboook to observe and assess your model's fairness using the following code:
+
+```python
+from fairlearn.widget import FairlearnDashboard
+
+# A_test containts your protected attributes (e.g., Age, Gender)
+# sensitive_feature_names containts your protected attributes names
+# y_true contains ground truth labels
+# y_pred contains prediction labels
+
+FairlearnDashboard(sensitive_features=A_test,
+                   sensitive_feature_names=['Gender', 'Age'],
+                   y_true=Y_test.tolist(),
+                   y_pred=[y_pred.tolist()])
+```
+
+
+Pre-mitigation visualization dashboard consists of procedural steps for configuration and evaluation of model fairness insights. In the configuration stage, you select your (1) protected (sensitive) attribute of interest (e.g., gender with three values of female, male, and non-binary), and (2) disparity metric that you would like to evaluate across different groups of your selected protected attribute (e.g., model precision). The evaluation stage uses these configurations to represent a detailed view of model's treatment across different subgroups of your selected protected attribute (e.g., model precision for females, model precision for males, model precision for people with non-binary gender). 
```

Comment:
'Pre-mitigation visualisation dashboard' …. users can include mitigated data too. Also, start with a definite article.

Should we _always_ use 'sensitive attribute' and never mention 'protected' ?

'to represent' -> 'to present'

Reply:
Agreed. I removed all protected attributes and I got rid of the discussion about pre-mitigation / post-mitigation. We talk about such workflows in our tutorials, so we can maybe figure a way how to surface that.